caikit/runtime/__main__.py

```diff
@@ -41,13 +39,23 @@ def interrupt(signal_, _stack_frame):
 
     # Start serving grpc server
     if get_config().runtime.grpc.enabled:
+        # Local
```

Comment:
NIT: Probably worth putting a comment here to explain why we're doing these locally. It might also be nice to have a `try/except` with a nice error message about installing the optional deps